test/fixtures/output/connect/offset/update-offset.golden

```diff
@@ -0,0 +1,8 @@
++------------+--------------------------------+
+| ID         | lcc-123                        |
+| Phase      | PENDING                        |
+| Message    | Alter offset process is        |
+|            | pending                        |
+| Applied At | 2024-04-02 08:23:33.000000123  |
```

Comment:
Applied At is also empty in pending phase. Please remove this as well.

Reply:
Similar for other examples of this type.

Reply:
This is just test